src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -134,7 +134,18 @@ private void UserSettingsService_OnSettingChangedEvent(object sender, SettingCha
 
 		private void HorizontalMultitaskingControl_Loaded(object sender, RoutedEventArgs e)
 		{
-			horizontalMultitaskingControl.DragArea.SizeChanged += (_, _) => SetRectDragRegion();
+			// WINUI3: bad workaround to be removed asap!
+			// SetDragRectangles() does not work on windows 10 with winappsdk "1.2.220902.1-preview1"
+			if (Environment.OSVersion.Version.Build >= 22000)
+			{
+				horizontalMultitaskingControl.DragArea.SizeChanged += (_, _) => SetRectDragRegion();
+			}
+			else
+			{
+				App.Window.AppWindow.TitleBar.ExtendsContentIntoTitleBar = false;
```

Comment:
`App.Window.AppWindow.TitleBar` is null on Windows 10

Reply:
Not anymore (appsdk 1.2 preview1)